src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -1272,6 +1282,7 @@ lVbRuntimePlus:
             Return New VisualBasicCommandLineArguments With
             {
                 .IsScriptRunner = IsScriptRunner,
+                .InteractiveMode = InteractiveMode Or (IsScriptRunner AndAlso sourceFiles.Count = 0),
```

Comment:
I think either a comment should be added (see the beginning of the thread), or an issue should be opened to add the comment later.